IORawData/CaloPatterns/src/Module.cc

```diff
@@ -1,7 +0,0 @@
-#include "IORawData/CaloPatterns/src/HcalPatternSource.h"
-#include "IORawData/CaloPatterns/src/HtrXmlPattern.h"
-
-#include "FWCore/Framework/interface/MakerMacros.h"
-
-DEFINE_FWK_MODULE(HcalPatternSource);
```

Comment:
Is it ok to get rid of those two `DEFINE_FWK_MODULE`, which are not ported into the source files?

Reply:
I see, yes, I should have moved that into the .cc files too, will do it now